### PR TITLE
[ADD][12.0] stock_generate_putaway_from_inventory: Add to 12.0

### PR DESCRIPTION
**Pull request by kevinkhao**

### CODE REVIEW

File: stock_generate_putaway_from_inventory/models/stock_inventory.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+                msg += "%s ," % el.name
+            msg = msg[:-2]
+        return msg
+
+    def action_generate_putaway_strategy(self):
+        missing_putaway = self.env["stock.location"]
+        not_validated = self.env["stock.inventory"]
+        for inventory in self:
+            try:
+                inventory._generate_putaway_strategy()
```
Comment (sebastienbeau) [inline stock_generate_putaway_from_inventory/models/stock_inventory.py]: I think we should keep this as simple a possible.
In the huge majority of case the user will generate putaway only for one inventory (most of customer create one inventory per year).
I think the code will be a lot simplify if we simplify raise directly the error.
So just do 
```
for inventory in self:
       inventory._generate_putaway_strategy()
```
And directly raise error in the method "_generate_putaway_strategy" then you can drop the handle_errors and so drop 16~20 line of code

**Review by sebastienbeau — CHANGES_REQUESTED** [review]

File: stock_generate_putaway_from_inventory/models/stock_inventory.py (hunk 1 above)
Comment (kevinkhao) [reply stock_generate_putaway_from_inventory/models/stock_inventory.py]: fixed at https://github.com/akretion/stock-logistics-warehouse/commit/f3159eba294a08566e4c752fe41cb6c6c7b1df9b

**Review by rousseldenis — APPROVED** [review]
Code review. LGTM

**Review by sebastienbeau — APPROVED** [review]

**Comment by sebastienbeau** [comment]
squash commit done let's do the last step /ocabot merge

**Pull request merged**